mixer/cmd/build.go

```diff
@@ -250,9 +247,6 @@ func init() {
 	buildChrootsCmd.Flags().BoolVar(&buildFlags.noSigning, "no-signing", false, "Do not generate a certificate to sign the Manifest.MoM")
 	buildChrootsCmd.Flags().BoolVar(&builder.UseNewChrootBuilder, "new-chroots", false, "EXPERIMENTAL: Use new implementation of build chroots")
 
-	// TODO: Remove or rewrite the message when we let yum go.
-	buildChrootsCmd.Flags().StringVar(&buildFlags.packager, "packager", "", "When using build-chroots, use passed packager program instead of yum (or dnf in Fedora >= 22)")
-
```

Comment:
Is there any use case for a packager other than yum and dnf? You could change to dnf without removing this feature, but if this won't be used, then It is a good idea to just remove it like you did.
Should we deprecate this to avoid breaking the API for now ?

Reply:
There wasn't any release of mixer with this flag, so we should be fine.